src/track/annotation/AnnotationTrack.tsx

```diff
@@ -238,6 +242,34 @@ export class AnnotationTrack extends TrackObject<AnnotationTrackModel, Annotatio
                 annotation.name.x = 5
                 annotation.name.relativeX = Math.max(annotation.gene.relativeX, 0);
 
+                //VALIS-8: prevent collision between annotation names
+                let annotationNameWidth = annotation.name.getComputedWidth();
```

Comment:
Can this be `const`? I don't see it reassigned anywhere

Reply:
Fixed